spec/models/annotation_spec.rb

```diff
@@ -8,6 +8,9 @@
       [CourseUserDatum.where(user: @students.first).first,
        CourseUserDatum.where(user: @students.second).first]
     end
+    after(:each) do
```

Comment:
nit: why not just use the shared context here?

Reply:
It's because the shared contexts exists for controllers (and lives within spec/controllers), we could do the same for models